Lib/fontbakery/specifications/post.py

```diff
@@ -49,10 +49,17 @@ def com_google_fonts_check_008(ttFonts):
 
 @check(id='com.google.fonts/check/015')
 def com_google_fonts_check_015(ttFont):
-  """Font has post table version 2?"""
-  if ttFont['post'].formatType != 2:
-    yield FAIL, ("Post table should be version 2 instead of {}."
+  """Font has correct post table version (2 for TTF, 3 for OTF)?"""
+  formatType = ttFont['post'].formatType
+  if 'glyf' in ttFont:
```

Comment:
Maybe like this(?):

```py
def com_google_fonts_check_015(ttFont, is_ttf):
  """Font has correct post table version (2 for TTF, 3 for OTF)?"""
  formatType = ttFont['post'].formatType
  if is_ttf:
  …
```

As `'glyf' in ttFont`  is pretty much the definition of `is_ttf`

Reply:
I made the suggested changes.